app/models/manageiq/providers/ibm_power_hmc/infra_manager/host.rb

```diff
@@ -1,6 +1,10 @@
 class ManageIQ::Providers::IbmPowerHmc::InfraManager::Host < ::Host
   include ManageIQ::Providers::IbmPowerHmc::InfraManager::MetricsCaptureMixin
 
+  supports :capture do
+    unsupported_reason_add(:capture, _("PCM not enabled for this Host")) unless advanced_settings.find { |s| s.name.eql?("pcm_enabled") }&.value.eql?("true")
```

Comment:
I think it'd be helpful to have this as a virtual_column in the host model...try this out:
```
def pcm_enabled
  !!advanced_settings.detect { |s| s.name == "pcm_enabled" }&.value
end
virtual_column :pcm_enabled, :type => :boolean, :uses => :advanced_settings
```

Reply:
Fixed it, I think

Reply:
:+1: yes looks great